server/controllers/content.js

```diff
@@ -0,0 +1,186 @@
+/* eslint-disable sort-keys */
+/* eslint-disable consistent-return */
+const HttpStatus = require('http-status-codes');
+const Content = require("../models/Content");
+
+const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR, CREATED, NOT_FOUND } = HttpStatus;
+
+// Create and Save a new Content
+exports.create = async(req, res) => {
+    try {
+        const { title, resourceType, tags, url, subjectId } = req.body;
+        // Validate request
+        if (!title) {
+            return res.status(BAD_REQUEST)
+                .send({
+                    status: "fail",
+                    data: { "title": "Content title is required" },
```

Comment:
We do not add quotations `""` around keys in javascript. Go through your files and fix this.